### PR TITLE
Create a stub of srreanu hometown file

### LINKED ISSUE #12: srreanu's recommendations for world
# World
- [x]  Create a branch
- [x]  Add a file
- [x] Commit the file
- [x] Open a pull request
- [x] Collaborate with my team
- [ ] Get a 👍 
- [ ] :shipit: Merge the pull request

@whitbeck @baryal @githubteacher @alephzed

### PR DESCRIPTION
**Pull request by srreanu**
Resolves #12 
## What do you want to know?
- Do you like specific types of food?
- Any types of activities you enjoy

@whitbeck @alephzed @divanathan @baryal @vvanaparthi @cheqmate @kkw-davita @vvanaparthi

### CODE REVIEW

**Comment by vvanaparthi** [comment]
Mexican, india

Boating horse riding

**Comment by cheqmate** [comment]
> Do you like specific types of food?

Edible types are the best. I don't much like the kinds that you cannot eat. After that my favorite type is delicious. I really like delicious edible food.

**Comment by whitbeck** [comment]
> Do you like specific types of food?

Spicy!!!

**Comment by srreanu** [comment]
Not sure if anyone realized, but my home town is World. You'll have lot of choices. 👯

File: place/ddsrreanu-world.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## World
+
+### Where to eat
+
+Many places
```
Comment (srreanu) [inline place/ddsrreanu-world.md:5]: Can you give me one specific example?

**Pull request merged**